packages/event-handler/tests/unit/appsync-graphql/Router.test.ts

```diff
@@ -0,0 +1,144 @@
+import { Router } from 'src/appsync-graphql/Router.js';
```

Comment:
Similar to previous comment, we should test cresting the Router without supplying a `logger` object to the constructor.